internal/install/packs/service_packs_fetcher.go

```diff
@@ -0,0 +1,111 @@
+package packs
+
+import (
+	"context"
+
+	"github.com/newrelic/newrelic-cli/internal/install/recipes"
+	"github.com/newrelic/newrelic-cli/internal/install/types"
+	log "github.com/sirupsen/logrus"
+)
+
+// ServicePacksFetcher is an implementation of the packsFetcher interface that
+// relies on the Nerdgraph-stitched o11y packs service to source its results.
+type ServicePacksFetcher struct {
+	client recipes.NerdGraphClient
+}
+
+// NewServicePacksFetcher returns a new instance of ServicePacksFetcher.
+func NewServicePacksFetcher(client recipes.NerdGraphClient) PacksFetcher {
+	f := ServicePacksFetcher{
+		client: client,
+	}
+
+	return &f
+}
+
+func (f *ServicePacksFetcher) FetchPacks(ctx context.Context, recipes []types.OpenInstallationRecipe) ([]types.OpenInstallationObservabilityPack, error) {
+	log.Tracef("FetchPacks.recipes: %+v", recipes)
+
+	packs := []types.OpenInstallationObservabilityPack{}
```

Comment:
Should this be generated by tutone and live in the client?

Reply:
Agreed that we should do this eventually. From last meeting, sounds like we're going to punt on this for now.

Reply:
Issue created: https://github.com/newrelic/newrelic-client-go/issues/748